### PR TITLE
Python backports for OpenSea

### PR DESCRIPTION
**Pull request by feuGeneA**
OpenSea is integrating with 0x v2 (for now), and they've hit a bug and a lack of needed functionality, both of which have already been fixed/implemented in the v3/development branch, so this PR is to backport that functionality for them.

### CODE REVIEW

**Comment by feuGeneA** [comment]
image

File: .circleci/config.yml [hunk 1]
```diff
@@ -166,7 +166,7 @@ jobs:
         working_directory: ~/repo
         docker:
             - image: nikolaik/python-nodejs:python3.7-nodejs8
-            - image: 0xorg/ganache-cli:2.2.2
+            - image: 0xorg/ganache-cli:4.4.0-beta.1
```
Comment (fabioberger) [inline .circleci/config.yml]: Was this intentional?

File: python-packages/sra_client/test/relayer/docker-compose.yml [hunk 2]
```diff
@@ -2,7 +2,7 @@
 version: '3'
 services:
     ganache:
-        image: "0xorg/ganache-cli:2.2.2"
+        image: "0xorg/ganache-cli:4.4.0-beta.1"
```
Comment (fabioberger) [inline python-packages/sra_client/test/relayer/docker-compose.yml]: Was this intentional?

File: .circleci/config.yml (hunk 1 above)
Comment (feuGeneA) [reply .circleci/config.yml]: Yes.  Adding in the test of `build_transaction()` ran me into a problem with Web3.py trying to do something with `eth_chainId`, and that wasn't a supported RPC method in the version of ganache-cli used in the 2.2.2 image.  The 4.4.0-beta.1 image has the updated ganache-cli, as well as the snapshot needed to support the tests, so I ran with it.

File: python-packages/sra_client/test/relayer/docker-compose.yml (hunk 2 above)
Comment (feuGeneA) [reply python-packages/sra_client/test/relayer/docker-compose.yml]: Yes, to stay in sync with the `.circleci/config.yml`

**Pull request merged**